docs/user/advanced.rst

```diff
@@ -1076,3 +1076,5 @@ coffee.
     r = requests.get('https://github.com', timeout=None)
 
 .. _`connect()`: https://linux.die.net/man/2/connect
+
+When the client has IPv6 support and the server has an IPv6 DNS record (AAAA), if the IPv4 connection fails the underlying (`urllib3`_) will automatically retry using IPv6, which may lead to an effective connection timeout of twice the specified time, so take that into account when setting the connection timeout. You can see a `Stackoverflow answer <https://stackoverflow.com/questions/33046733/force-requests-to-use-ipv4-ipv6/46972341#46972341>`_ to get around the problem.
```

Comment:
Let's make this far clearer and not link to stackoverflow.

Something like

```suggestion
The connect timeout applies to each connection attempt to an IP address. If multiple addresses exist for a domain name, we can try each one. If those fail before the connection timeout is reached, urllib3 will keep trying additional IP addresses. In some cases - like IPv6 (AAAA record) - the IPv6 connection can be slower to fail. 

This phenomenon can lead to an appearance of requests ignoring or multiplying the connect timeout. Based on your specific situation, you may consider extreme solutions like patching the standard library to avoid returning IPv6 records for domain names. If done poorly, this can lead to additional issues for Requests which maintainers will not help you debug.
```

Reply:
@sigmavirus24
Thank you for your comment.
I've added a more detailed explanation than the previous commit.

Reply:
Your comment was very helpful. thanks you.
I tried it on multiple translation sites and confirmed that the meaning did not change, but the grammar may be wrong

Reply:
We have multiple volunteers who translate the documentation for us. We don't need to put it through documentation sites. Further, IPv6 is _one_ very specific case. My revision calls out that this is possible _regardless_ of IPv4 or IPv6.

Reply:
@sigmavirus24
I corrected the part you pointed out.
Sorry for the inconvenience.